BraintreeCore/src/test/java/com/braintreepayments/api/PaymentMethodNonceUnitTest.java

```diff
@@ -29,6 +31,8 @@ public void constructor_parsesPayPalFromPlainJSONObject() throws JSONException {
         assertFalse(nonce.isDefault());
         assertEquals(PaymentMethodType.PAYPAL, nonce.getType());
         assertEquals("aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee", nonce.getString());
+        assertEquals("PayPal", nonce.getTypeLabel());
+        assertEquals("with email paypalaccount@example.com", nonce.getDescription());
```

Comment:
I know these are just unit tests, but it's confusing to me that we say "with email paypalaccount@example.com" here. Can we just make the description "paypalaccount@example.com"?

Reply:
The test assertions should be good now!